eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -74,6 +74,44 @@ void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOExcep
         );
     }
 
+    @Disabled
+    @Test
+    void returnsFromCacheButTimesSaveAndExecuteDifferent(@TempDir final Path tmp)
+        throws IOException, InterruptedException {
+        final XML program = OptCachedTest.program(ZonedDateTime.now());
+        Thread.sleep(70_000);
```

Comment:
@Yanich96 It is a ["Wait and See"](https://stackoverflow.com/a/339339/10423604) anti-pattern